jdaviz/core/template_mixin.py

```diff
@@ -2681,7 +2684,8 @@ def add_scatter(self, label, x=[], y=[], xnorm=False, ynorm=False, **kwargs):
                               colors=kwargs.pop('color', kwargs.pop('colors', 'gray')),
                               **kwargs)
 
-    def add_bins(self, label, sample=[1], bins=1, density=True, **kwargs):
+    def add_bins(self, label, sample=[0], bins=2, density=True, **kwargs):
+        # NOTE: initializing with bins=1 breaks the figure until a resize event
```

Comment:
@pllim @javerbukh - this was the actual fix for the tray-resize issue.  I kept a few other small changes from attempts to fix this that I think will still be more efficient, but I don't expect them to be very controversial 🤞

Reply:
My approval still stands. I don't see how this is going to break anything new. Thanks!

Reply:
My approval stills stands as well!